Source/WarpX.cpp

```diff
@@ -735,11 +735,18 @@ WarpX::ReadParameters ()
         }
 
         pp.query("current_correction", current_correction);
-        pp.query("v_galilean", m_v_galilean);
         pp.query("v_comoving", m_v_comoving);
         pp.query("do_time_averaging", fft_do_time_averaging);
 
-        // Scale the velocity by the speed of light
+        // Check whether the default Galilean velocity should be used
+        bool use_default_v_galilean=false;
+        pp.query("use_default_v_galilean", use_default_v_galilean);
+        if (use_default_v_galilean) {
+            v_galilean[2] = -std::sqrt(1-1./(gamma_boost*gamma_boost));
```

Comment:
I think that the missing `m_` prefix is causing the various CI builds to fail (while the `_rt` should fix implicit conversions):
```suggestion
            m_v_galilean[2] = -std::sqrt(1._rt-1._rt/(gamma_boost*gamma_boost));
```